__tests__/app.test.js

```diff
@@ -325,4 +325,135 @@ describe("GET /api/users",() => {
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    review_id: expect.any(Number),
+                    comment_count: expect.any(String)
+                })
+            )
+           })
+        })
+    })
+    test("sorting by column not in table will return 400 and helpful error message", () => {
+        return request(app).get('/api/reviews?sort_by=banana')
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({msg:"sort_by column not found in database"})
+        })
+    })
+    test("accepts order query, sorts data appropriately into DESC or ASC", () => {
+        return request(app).get('/api/reviews?order=asc')
+        .expect(200)
+        .then(({body:{reviews}}) => {
+            expect(reviews).toBeSortedBy("created_at",{ascending:true})
+        })
+    })
+    test("incorrect order value results in 400 and useful error message", () => {
+        return request(app).get('/api/reviews?order=banana')
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({msg: "order by query invalid"})
+        })
+    })
+    test("queries not on the list get ignored and result in default behaviour, 200", () => {
+        return request(app).get('/api/reviews?banana=true')
+        .expect(200)
+        .then(({body:{reviews}}) => {
+            expect(reviews).toHaveLength(13)
+            expect(reviews).toBeSortedBy("created_at",{descending:true})
+            reviews.forEach((review) => {
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        title: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_img_url: expect.any(String),
+                        category: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        review_id: expect.any(Number),
+                        comment_count: expect.any(String)
+                    })
+                )
+            })
+        })
+    })
+    test("can handle multiple valid queries simultaneously and give correct result", () => {
+        return request(app).get('/api/reviews?category=social+deduction&sort_by=owner&order=asc')
+        .expect(200)
+        .then(({body:{reviews}}) => {
+            expect(reviews).toHaveLength(11)
+            expect(reviews).toBeSortedBy("owner",{ascending:true})
+            reviews.forEach((review) => {
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        title: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_img_url: expect.any(String),
+                        category: expect.any(String),
```

Comment:
Could we check the category is social deduction for this expect to verify the response is as expected.